frontend/ui/widget/bookstatuswidget.lua

```diff
@@ -91,6 +92,14 @@ function BookStatusWidget:init()
                 seqtext = "any key", doc = "close dialog" }
         }
     end
+    if Device:isTouchDevice() then
+        self.ges_events.Swipe = {
+            GestureRange:new{
+                ges = "swipe",
+                range = function() return self.dimen end,
```

Comment:
More contained ? What do you mean ?
a) smaller available area to start the swipe ?
b) limit the swipe length ?

Given that swipe north and south were not used (ignored, or fallback to trigger full refresh) on all these widgets, it feels a bit unnecessary to limit them in any way.

Reply:
A limited swipe length is (almost?) always stupid. A smaller starting area is not.

> Given that swipe north and south were not used (ignored, or fallback to trigger full refresh) on all these widgets, it feels a bit unnecessary to limit them in any way.

But you don't want to make closing super simple to perform accidentally.

I'll have to play around with it some more but so far I'm afraid I don't like it very much.

Reply:
> But you don't want to make closing super simple to perform accidentally.

I do :) I did that because closing these was actually difficult (meaning, you need to use 2 hands, or some balance if you keep the device one handed from the bottom right, to reach the button at top right with a finger from that hand, often needing a few tries to hit that button :)

And for the impacted widget, it's mostly something you consult, grab the info (description, book title, where you are in the TOC, see what chapters/topics are coming), and quickly want to get out of it. And it's easy to get back to it, now that menu are kept open - except in the rare cases where you are many page away from the first one, where you would need to browse all of them again.
But actually, it's easier to get a biger accident with Tap : I often do, tap by error (lazyness, or bounce) on a book in History, or an entry in TOC, and 1) you're out of the widget 2) you're opening a book you didn't want to or are thrown away to some other page (and that hopefully didn't make us change the Tap to a Hold :) A small south swipe needs more willingness from the use than a Tap.

But yes, try it.
If you still feel it's dangerous, I'm ok to require a minimal swipe length of 1/4th of screen, which could still be done securely with one hand (1/3th at max - 1/2th feels already less secure).

Shall I change the south+north to only south? (Can't think of any other action on these widgets we one day would like to assign to swipe north thus.)

Reply:
It's better not to have to break acquired behaviors later, so yes, just south. :-)

Reply:
Should I _reserve_ north (so only diagonal swipe does full refresh, and people stop acquiring the habit of swiping north fot that) with: ?
```lua
elseif ges_ev.direction == "north"  then
  do end
else
    -- trigger full refresh
end
```
Or just fallback swipe north to trigger full refresh? (in other non full screen widgets, mostly only diagonal works).

Reply:
I'm not even sure if I knew that triggered refresh. It definitely seems like a weird and confusing thing to allow.

Reply:
OK, with last commits, trigger full refresh is limited to diagonal swipe, and swipe north does nothing.
(I hope you are not discussing the use of diagonal swipe to trigger full refresh :)